File: Android/Notelimites/app/src/main/java/com/burocreativo/notelimites/io/models/token/SignInResult.java

package com.burocreativo.notelimites.io.models.token;

import com.google.gson.annotations.SerializedName;

@SuppressWarnings("unused")
public class SignInResult {

    @SerializedName("auth_token")
    private String mAuthToken;
    @SerializedName("email")
    private String mEmail;

    public String getAuthToken() {
        return mAuthToken;
    }

    public void setAuthToken(String auth_token) {
        mAuthToken = auth_token;
    }

    public String getEmail() {
        return mEmail;
    }

    public void setEmail(String email) {
        mEmail = email;
    }

}

File: No te limites/No te limites/SignInResult.swift
//
//  SignInResult.swift
//  No te limites
//
//  Created by Griselda Epifanio on 12/8/16.
//  Copyright © 2016 Jose Epifanio. All rights reserved.
//

import Foundation

class SignInResult
{
    private var mAuthToken:String = ""
    private var mEmail:String = ""
    
    //Seters
    public func setAuthToken(authToken: String) {
        self.mAuthToken = authToken
    }
    
    public func setEmail(email: String) {
        self.mEmail = email
    }
    
    //Geters
    public func getAuthToken()-> String {
        return self.mAuthToken
    }
    
    public func getEmail()-> String {
        return self.mEmail
    }
    
}
